source-map-support.js

```diff
@@ -88,12 +88,14 @@ retrieveFileHandlers.push(function(path) {
       if (xhr.readyState === 4 && xhr.status === 200) {
         contents = xhr.responseText;
       }
-    } else if (fs.existsSync(path)) {
+    } else {
```

Comment:
both usages of `fs.readFileSync` are already inside a `try/catch` block, it's safe to just let them try the read - it gets swallowed anyway